src/status_im2/common/confirmation_drawer/view.cljs

```diff
@@ -34,9 +34,9 @@
                     profile-picture name]} context
             id                             (or chat-id public-key)
             display-name                   (or
-                                            name
                                             (when-not group-chat
-                                              (rf/sub [:contacts/contact-name-by-identity id])))
+                                              (rf/sub [:contacts/contact-name-by-identity id]))
```

Comment:
This is the kind of thing we should have a subscription for, so that we can avoid code duplication to handle the logic for display names.

Reply:
can we either add this as part of this pr, or create a follow up issue so it can be addressed?